client/src/__tests__/EditEntry.test.jsx

```diff
@@ -0,0 +1,20 @@
+import { render, screen } from '@testing-library/react';
+import { BrowserRouter as Router } from 'react-router-dom';
+
+import EditEntry from '../components/EditEntry';
+
+test('buttons render', () => {
+  render(
+    <Router>
+        <EditEntry />
```

Comment:
This is more of a general comment. Do you need to pass down props to this component to get it to work? Think about it!